sdk/openai/openai/review/openai-types.api.md

```diff
@@ -158,12 +121,6 @@ export interface ContentFilterCitedDetectionResultOutput {
     URL?: string;
```

Comment:
should this be lowercase `url`?

Reply:
I see it as uppercase in
https://github.com/Azure/azure-rest-api-specs/blob/fb64eaa0dfc7ee8abc8e10369495729d5b3c1cc5/specification/cognitiveservices/data-plane/AzureOpenAI/inference/stable/2024-10-21/inference.json#L791